doc/lstm.txt

```diff
@@ -22,23 +184,33 @@ The LSTM implementation can be found in the two following files :
 
 * `imdb.py <http://deeplearning.net/tutorial/code/imdb.py>`_ : Secondary script. Handles the loading and preprocessing of the IMDB dataset.
 
-Data
-====
+After downloading both the scripts, downloading the data and
+putting all those files in the same folder, the user can run the code by
```

Comment:
Do not tell downloading the data.

You can add after the execution command, that it will automatically download the data.